service-python/assessclaimdc7101/src/lib/continuous_medication.py

```diff
@@ -37,7 +37,7 @@ def filter_mas_medication(event):
     medication_with_date = []
     medication_without_date = []
     medication_two_years = []
-    date_of_claim_date = datetime.strptime(event["dateOfClaim"], "%Y-%m-%d").date()
+    date_of_claim_date = datetime.strptime(event["claimSubmissionDateTime"], "%Y-%m-%dT%H:%M:%SZ").date()
```

Comment:
In case the date is missing, or is not in the expected format, this is going to throw an exception and terminate processing. Is this the desired result?

Reply:
@jbregstone, is this where we need to pass `""` or `No associated date` or is this unrelated?

Reply:
@anthonyvelazquez It is a different date that is not on the PDF as far as I can tell.

Reply:
@dimitri-amida I will add some exception handling in case the parsing fails.